extension/packages/nextjs/components/Header.tsx.args.mjs

```diff
@@ -1,3 +1,19 @@
+export const menuIconImports = `import { BanknotesIcon, EyeIcon, UserIcon } from "@heroicons/react/24/outline";`;
+export const menuObjects = `{
```

Comment:
not sure what's the problem for now but when trying `yarn vercel` I'm getting this error on build
<img width="1488" alt="image" src="https://github.com/user-attachments/assets/4fd0dc72-1153-454c-9798-afa31c559591" />

error message:
`Type error: Type '({ label: string; href: string; } | { label: string; href: string; icon: Element; } | undefined)[]' is not assignable to type 'HeaderMenuLink[]'`

Reply:
oh, I resulting code in the header is
<img width="460" alt="image" src="https://github.com/user-attachments/assets/9af7e96f-74e9-46ac-8b5d-aa53030f7202" />

Looks like you need to remove that `,` after Oracle object